packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import { useHistory } from 'react-router-dom';
 import { useActiveTradingAccount, useResetVirtualBalance } from '@deriv/api';
 import { Provider } from '@deriv/library';
-import { Button, Text } from '@deriv/quill-design';
+import { Text } from '@deriv/quill-design';
```

Comment:
Cant we use Text from deriv/ui as we are using Button from it?

Reply:
There's another user story for that. Plus i am changing button cause of the weird left align from qull-design